clang-tools-extra/include-cleaner/lib/Record.cpp

```diff
@@ -232,6 +232,17 @@ class PragmaIncludes::RecordPragma : public PPCallbacks, public CommentHandler {
   void checkForExport(FileID IncludingFile, int HashLine,
                       std::optional<Header> IncludedHeader,
                       OptionalFileEntryRef IncludedFile) {
+    auto AddExport = [&] {
```

Comment:
Can you inline this lambda? I think the code is correct, but I found it a bit hard to reason about the change.

Reply:
this was preparation for a follow-up change, but I guess it's better to do it there.